alexandra/app.py

```diff
@@ -107,6 +113,10 @@ def intent(self, intent_name):
             def foo_bar_baz_intent(slots, session):
                 pass
 
+            @alexa_app.intent('BazFooBar')
```

Comment:
Could you also update the docstring text above to mention that 3 arguments are also supported, and what the `ids` parameter will contain?